src/App.tsx

```diff
@@ -1,11 +1,23 @@
-import "./App.css";
+import { useState } from 'react';
+import './styles/index.css';
+import styles from './App.module.css';
+import CardNumberSection from './components/CardNumberSection/CardNumberSection';
+import CardExpirationSection from './components/CardExpirationSection/CardExpirationSection';
+import Card from './components/Card/Card';
+import CvcSection from './components/CvcSection/CvcSection';
+
+export default function App() {
+  const [cardNumbers, setCardNumbers] = useState<string[]>(['', '', '', '']);
+  const [cardLogo, setCardLogo] = useState<'visa' | 'master' | ''>('');
```

Comment:
```suggestion
  type CardLogo = 'visa' | 'master';
  const [cardLogo, setCardLogo] = useState<CardLogo>('');
```
`'visa' | 'master' | ''` 타입이 다른 곳에서도 사용되는데 분리해보는건 어떨까요??🙃

그리고  타입에 빈 문자열이 포함되어야 할까요??🤔